test/unit/api/opterm_black.h

```diff
@@ -55,3 +60,157 @@ void OpTermBlack::testIsNull()
   x = d_solver.mkOpTerm(BITVECTOR_EXTRACT_OP, 31, 1);
   TS_ASSERT(!x.isNull());
 }
+
+void OpTermBlack::testMakeDivisibleOp()
```

Comment:
This should actually go into solver_black, just add it in this function: https://github.com/CVC4/CVC4/blob/master/test/unit/api/solver_black.h#L445

Reply:
Oops, fixed with 4d91687f33ba4186d7230e307126221b1e62d86c. Thanks for the pointer.